sopel/plugins/handlers.py

```diff
@@ -64,6 +61,26 @@
     from types import ModuleType
 
 
+class PluginMetaDescription(TypedDict):
```

Comment:
I was wondering if it would make sense to do this while looking at #2498, I'm in favor of it! :grin: